pkg/volume/util/operationexecutor/operation_executor_test.go

```diff
@@ -224,3 +354,49 @@ func getTestPodWithGCEPD(podName, pdName string) *v1.Pod {
 		},
 	}
 }
+
+func isOperationRunSerially(ch, quit chan interface{}) bool {
+	defer close(quit)
+	numOperationsStarted := 0
+loop:
+	for {
+		select {
+		case <-ch:
+			numOperationsStarted++
+			if numOperationsStarted > 1 {
+				return false
+			}
+		case <-time.After(5 * time.Second):
```

Comment:
Let's drop this to 30 seconds. Blocking a unit test for 5 minutes on every execution seems like a bad idea. It's no ideal, but...